data/json/items/gun/9mm.json

```diff
@@ -1,43 +1,21 @@
 [
-  {
-    "id": "90two",
```

Comment:
This needs a migration, see https://github.com/CleverRaven/Cataclysm-DDA/blob/a2e692099ce1d4f3c21354977b2d02e66c315fa8/data/json/items/migration.json#L1156-L1161 for an example.